addons/interaction/functions/fnc_canPassMagazine.sqf

```diff
@@ -12,19 +12,21 @@
  * Unit can pass magazine <BOOL>
  *
  * Example:
- * [_player, _target, "arifle_MX_F"] call ace_interaction_fnc_canPassMagazine
+ * [player, cursorObject, "arifle_MX_F"] call ace_interaction_fnc_canPassMagazine
  *
  * Public: No
  */
-params ["_player", "_target", "_weapon"];
 
-if (!GVAR(enableMagazinePassing)) exitWith {false};
-if (_weapon isEqualTo "" || {!(_target call EFUNC(common,isAwake))}) exitWith {false};
-if ((!isNull objectParent _target) && {(vehicle _target) != (vehicle _player)}) exitWith {false};
+params ["_player", "_target", "_weapon"];
 
-private _compatibleMags = [_weapon] call CBA_fnc_compatibleMagazines;
+GVAR(enableMagazinePassing) &&
+{_weapon != ""} &&
+{_target call EFUNC(common,isAwake)} &&
+{(objectParent _target) isEqualTo (objectParent _player)} &&
+{
+    private _compatibleMags = compatibleMagazines _weapon;
```

Comment:
not a problem, just noting that this will include 2nd muzzle mags (like 40mm for a glrifle) compared to the cba func